deno/rest/v10/mod.ts

```diff
@@ -844,6 +844,258 @@ export const Routes = {
+	defaultUserAvatar(userDiscriminator: 0 | 1 | 2 | 3 | 4 | 5) {
+		return `/embed/avatars/${userDiscriminator}.png` as const;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/avatars/{user.id}/{user.avatar}.{png|jpeg|webp|gif}`
+	 *
+	 * As this route supports GIFs, the hash will begin with `a_` if it is available in GIF format
+	 *
+	 * This route supports the extensions: PNG, JPEG, WebP, GIF
+	 */
+	userAvatar(userId: Snowflake, userAvatar: string, format: UserAvatarFormat) {
+		return `/avatars/${userId}/${userAvatar}.${format}` as const;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/guilds/{guild.id}/users/{user.id}/{guild_member.avatar}.{png|jpeg|webp|gif}`
+	 *
+	 * As this route supports GIFs, the hash will begin with `a_` if it is available in GIF format
+	 *
+	 * This route supports the extensions: PNG, JPEG, WebP, GIF
+	 */
+	guildMemberAvatar(guildId: Snowflake, userId: Snowflake, memberAvatar: string, format: GuildMemberAvatarFormat) {
+		return `/guilds/${guildId}/users/${userId}/avatars/${memberAvatar}.${format}` as const;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/app-icons/{application.id}/{application.icon}.{png|jpeg|webp}`
+	 *
+	 * This route supports the extensions: PNG, JPEG, WebP
+	 */
+	applicationIcon(applicationId: Snowflake, applicationIcon: string, format: ApplicationIconFormat) {
+		return `/app-icons/${applicationId}/${applicationIcon}.${format}` as const;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/app-icons/{application.id}/{application.cover_image}.{png|jpeg|webp}`
+	 *
+	 * This route supports the extensions: PNG, JPEG, WebP
+	 */
+	applicationCover(applicationId: Snowflake, applicationCoverImage: string, format: ApplicationCoverFormat) {
+		return `/app-icons/${applicationId}/${applicationCoverImage}.${format}` as const;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/app-icons/{application.id}/{application.asset_id}.{png|jpeg|webp}`
+	 *
+	 * This route supports the extensions: PNG, JPEG, WebP
+	 */
+	applicationAsset(applicationId: Snowflake, applicationAssetId: string, format: ApplicationAssetFormat) {
+		return `/app-icons/${applicationId}/${applicationAssetId}.${format}` as const;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/app-assets/{application.id}/achievements/{achievement.id}/icons/{achievement.icon}.{png|jpeg|webp}`
+	 *
+	 * This route supports the extensions: PNG, JPEG, WebP
+	 */
+	achievementIcon(
+		applicationId: Snowflake,
+		achievementId: Snowflake,
+		achievementIconHash: string,
+		format: AchievementIconFormat,
+	) {
+		return `/app-assets/${applicationId}/achievements/${achievementId}/icons/${achievementIconHash}.${format}` as const;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/app-assets/710982414301790216/store/{sticker_pack.banner.asset_id}.{png|jpeg|webp}`
+	 *
+	 * This route supports the extensions: PNG, JPEG, WebP
+	 */
+	stickerPackBanner(stickerPackBannerAssetId: Snowflake, format: StickerPackBannerFormat) {
+		return `app-assets/710982414301790216/store/${stickerPackBannerAssetId}.${format}` as const;
```

Comment:
```suggestion
		return `/app-assets/710982414301790216/store/${stickerPackBannerAssetId}.${format}` as const;
```

Reply:
Could the app id also be a constant somewhere (710982414301790216)? Something like `StickerPackApplicationId` maybe

Reply:
This const had to be placed on the top of `CDNRoutes`